ui/imports/shared/popups/DownloadPage.qml

```diff
@@ -0,0 +1,88 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+
+import utils 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import "../panels"
+import "."
+
+// TODO: replace with StatusModal
+Rectangle {
+    id: root
+
+    property string currentVersion: "0.0.0"
+    property string newVersion: "0.0.0"
+    property bool newVersionAvailable: true
+    property string downloadURL: "https://github.com/status-im/status-desktop/releases/latest"
```

Comment:
isn't this something we should rely on the BE for just in case this link changes in the future?

Reply:
That's just a default value in case the BE doesn't return a url (it should always return one in normal cases)